### ISSUE TITLE
Segfaults on launch on openSUSE

### ISSUE THREAD

**Issue by lgbaldoni** [clangen/musikcube#139]
Before you ask, yes I saw https://github.com/clangen/musikcube/issues/122 and https://github.com/clangen/musikcube/issues/108, but just to make things clear: what version of ncurses is required?

```
Thread 1 "musikcube" received signal SIGSEGV, Segmentation fault.
                                                                                                 0x00007ffff7113547 in _nc_printf_string_sp (sp=0x0, fmt=fmt@entry=0xa00778 "settings_first_run_dialog_title", 
    ap=ap@entry=0x7fffffffcdb8) at ../ncurses/base/safe_sprintf.c:251
251             if (screen_lines(SP_PARM) > MyRows || screen_columns(SP_PARM) > MyCols) {
```

**Comment by clangen** [comment]
I've tested musikcube with ncurses5 (Debian) and ncurses6 (Arch) -- it seems to work fine with both. But you must make sure you link against the "wide-character" libs, otherwise things will likely crash.

I just got done installing an OpenSUSE Tumbleweed VM. After enabling the "packman" repo, I did the following:

```sudo zypper install libfaad-devel libcurl-devel libmicrohttpd-devel boost_1_64-devel cmake ncurses5-devel libogg-devel libvorbis-devel flac-devel liblame3-devel pulseaudio libpulse-devel alsa-devel zlib-devel``` 

Then I cloned the sources and built and ran without issue. It seems to be fine here.

What version of OpenSUSE are you running? And are you sure you are compiling the latest `master` branch? Those two tickets you referenced are fixed, but the fixes have not been released yet, they are still hanging out in the `master` branch.
